src/__tests__/unit.test.tsx

```diff
@@ -0,0 +1,41 @@
+import { lazyForPaint } from '..';
```

Comment:
So generally unit tests should live closer to the actual unit, and top level tests folder are either reserved for integration style tests, or files that are in the root directory. Could this file be moved to the `lazy` directory under its own `__tests__` folder, or introduce a `.test.tsx` pattern and place it appropriately?

Reply:
I was just following the convention established here. Happy to move it if that's what we want to do